docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -140,6 +150,30 @@ main().catch(err => {
     process.exit(1);
 });
 ----
+def entry_added(event):
+    print("Entry added with key: %s, value: %s" % (event.key, event.value))
+
+
+def entry_removed(event):
+    print("Entry removed with key:", event.key)
+
+
+def entry_updated(event):
+    print(
+        "Entry updated with key: %s, old value: %s, new value: %s"
+        % (event.key, event.old_value, event.value)
+    )
+
+
+client = hazelcast.HazelcastClient()
+
```

Comment:
Could you remove the excess blank lines?

Reply:
Done.